files/mcloud/node_modules/@appium/base-driver/build/lib/protocol/protocol.js

```diff
@@ -393,128 +393,131 @@ function buildHandler(app, method, path, spec, driver, isSessCmd) {
         }
         catch (err) {
 
-            async function updateDeviceStatus(token, udid, note) {
+            // token: string - auth token from stf 
+            // udid: string -  unique id of device 
+            // note: string - comment with error for device which would shows in stf
+            // isMakedDeviceUnhealthy: boolean - if true device status will change to Unhealthy, if false comment will added 
+            async function updateDeviceStatus(token, udid, note, isMakeDeviceUnhealthy) {
 
                 // if token or udid not exist, skip function 
                 if (token == "" || udid == "") {
                     console.log(`[MCLOUD] Device status didi not change in STF`)
-                    return; 
+                    return;
                 }
-            
+
                 const url = `${process.env.STF_API_URL}/devices/${udid}`;
-                
+
                 const headers = {
-                  'Authorization': `Bearer ${token}`,
-                  'Content-Type': 'application/json'
-                };
-              
-                const data = {
-                    device: {
-                        status: 'Unhealthy',
-                        note: note
-                    }
+                    'Authorization': `Bearer ${token}`,
+                    'Content-Type': 'application/json'
                 };
-              
+
+
+                const data = isMakeDeviceUnhealthy ? { device: { status: 'Unhealthy', note: note } } : { device: { note: note } }
+
                 try {
-                  const response = await fetch(url, {
-                    method: 'PUT',
-                    headers: headers,
-                    body: JSON.stringify(data)
-                  });
-              
-                  if (!response.ok) {
-                    throw new Error(`[MCLOUD] HTTP Error ${response.status}`);
-                  }
-              
-                  const responseData = await response.json();
-                  console.log('[MCLOUD] Successful device status change response:', responseData);
-              
-                  return responseData;
+                    const response = await fetch(url, {
+                        method: 'PUT',
+                        headers: headers,
+                        body: JSON.stringify(data)
+                    });
+
+                    if (!response.ok) {
+                        throw new Error(`[MCLOUD] HTTP Error ${response.status}`);
+                    }
+
+                    const responseData = await response.json();
+                    console.log('[MCLOUD] Successful device status change response:', responseData);
```

Comment:
after ":" gramma recommend to add space so concatenated response easear to read :)
inspect in every your lines